swift_scality_backend/server.py

```diff
@@ -60,63 +70,134 @@ def setup(self, conf):
         if sproxyd_read_timeout is not None:
             kwargs['read_timeout'] = float(sproxyd_read_timeout)
 
-        self._filesystem = SproxydClient(sproxyd_urls, logger=self.logger, **kwargs)
         self._diskfile_mgr = swift_scality_backend.diskfile.DiskFileManager(conf, self.logger)
 
+        self._conn_timeout = float(sproxyd_conn_timeout) \
```

Comment:
This will not work. If sproxyd_conn_timeout is not set in the object-server.conf, then `self._conn_timeout` will be `None` which means 'no timeout' for the sproxyd client. (see comment L61)